test/patches/convertInlinePoliciesToManaged.test.ts

```diff
@@ -41,6 +48,26 @@ describe('Updating Resource Types', () => {
     appTemplate.resourceCountIs('AWS::IAM::Policy', 0);
     appTemplate.resourceCountIs('AWS::IAM::ManagedPolicy', 1);
   });
+
+  test('Inline Direct in Role', () => {
+    new ecr.Repository(stack, 'Repo', {
+      autoDeleteImages: true,
+      removalPolicy: RemovalPolicy.DESTROY,
+    });
+
+    Aspects.of(app).add(new ConvertInlinePoliciesToManaged());
+    app.synth();
+
+    const appTemplate = Template.fromStack(stack);
+    console.log(appTemplate);
+    let roles = appTemplate.findResources('AWS::IAM::Role');
+    for (const k in roles) {
+      const role = roles[k];
+      console.log(role);
+    }
+    expect(false).toBe(true);
```

Comment:
This test is failing

```
  ● Updating Resource Types › Inline Direct in Role

    expect(received).toBe(expected) // Object.is equality

    Expected: true
    Received: false

      66 |       console.log(role);
      67 |     }
    > 68 |     expect(false).toBe(true);
         |                   ^
      69 |   });
      70 |
```

This would never work - as you are just saying the boolean false should be true